rust/operator-binary/src/backend/dynamic.rs

```diff
@@ -128,6 +128,14 @@ pub async fn from_class(
             )
             .await?,
         ),
+        crd::SecretClassBackend::CertManager(crd::CertManagerBackend {
+            issuer,
+            default_certificate_lifetime,
+        }) => from(super::CertManager {
```

Comment:
`Box<Dynamic>` is also `impl SecretBackend`, so that would conflict with the blanket `impl<T> From<T> for T`.

Pretty sure the orphan rule would also prevent us from implementing `From` on `Box`.

Reply:
Ah of course... I was considering the orphan rule, but totally didn't even think about `From` :man_facepalming: 

As you were :)

Reply:
I guess since the blanket impl exists, maybe we can still remove `from` and do:

```rust
super::CertManager { ... }.into() // might need the type hint though.
```

_Just throwing ideas. `from` is convenient._